deploy/examples/cluster.yaml

```diff
@@ -19,9 +19,9 @@ spec:
     # v15 is octopus, and v16 is pacific.
     # RECOMMENDATION: In production, use a specific version tag instead of the general v14 flag, which pulls the latest release and could result in different
```

Comment:
v14?

Reply:
The https://hub.docker.com/r/ceph/ceph/tags/ ul latest tag is v.16 so updated to it, will see when the v17 tag is available.

Reply:
The Docker Hub is not updated anymore (for a few months already), all new images are published to quay.io only.